### PR TITLE
Allow vm.args to be set with env var

### PR DESCRIPTION
**Pull request by josevalim**

### CODE REVIEW

**Comment by jesseshieh** [comment]
@josevalim I really do not understand how you are able to get things done so quickly! This looks good to me.

File: lib/mix/lib/mix/tasks/release.init.ex [hunk 1]
```diff
@@ -237,6 +241,7 @@ defmodule Mix.Tasks.Release.Init do
     if not defined RELEASE_COOKIE (set /p RELEASE_COOKIE=<!RELEASE_ROOT!\releases\COOKIE)
     if not defined RELEASE_NODE (set RELEASE_NODE=!RELEASE_NAME!@127.0.0.1)
     if not defined RELEASE_TMP (set RELEASE_TMP=!RELEASE_ROOT!\tmp)
+    if not defined RELEASE_VM_ARGS (set RELEASE_TMP=!RELEASE_ROOT!\releases\!RELEASE_VSN!\vm.args)
```
Comment (jesseshieh) [inline lib/mix/lib/mix/tasks/release.init.ex]: I think `RELEASE_TMP` here should be `RELEASE_VM_ARGS`

Comment (josevalim) [reply lib/mix/lib/mix/tasks/release.init.ex]: Perfect, that should fix the build.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**